test/models/work_test.rb

```diff
@@ -0,0 +1,137 @@
+require "test_helper"
+
+describe Work do
+  let(:work) { works(:harrypotter)  }
+
+  it "must be valid" do
+    expect(work).must_be :valid?
+  end
+
+  it 'has required fields' do
+    fields = [:title, :creator, :description, :publication_year, :category]
+
+    fields.each do |field|
+      expect(work).must_respond_to field
+    end
+  end
+
+  describe 'Relationships' do
+    it 'can have many votes' do
+      # Arrange (done with let)
+      work = works(:harrypotter)
+      vote1 = votes(:voteone)
+      # Act
+      vote = work.votes
+
+      # Assert
+      expect(vote).must_be_instance_of Vote
+      expect(vote.id).must_equal work.votes_id
+    end
+  #
+  #   it 'can have many users' do
+  #     # Arrange, did with let
+  #
+  #     # Act
+  #     book.genres << Genre.first
+  #     genres = book.genres
+  #
+  #
+  #     # Assert
+  #     expect(genres.length).must_be :>=, 1
+  #     genres.each do |genre|
+  #       expect(genre).must_be_instance_of Genre
+  #     end
+  #   end
+  end
+
+  describe 'validations' do
+    it 'must have a title' do
+      # Arrange
+      work = works(:harrypotter)
+      work.title = nil
+
+      # Act
+      #valid = book.valid?
+      valid = work.save
+
+      # Assert
+      expect(valid).must_equal false
+      expect(work.errors.messages).must_include :title
+      expect(work.errors.messages[:title]).must_equal ["can't be blank"]
+    end
+
+    it 'cannot have category other than movie, album or book' do
+      # Arrange
+      work = works(:harrypotter)
+
+      [nil, "art", 123, "indie"].each do |cat|
+        work.category = cat
+        valid = work.save
+        expect(valid).must_equal false
+        expect(work.errors.messages).must_include :category
+      end
+    end
+
+    it 'must have category movie, album or book' do
+      # Arrange
+      work = works(:harrypotter)
+
+      ["movie", "album", "book"].each do |cat|
+        work.category = cat
+        valid = work.save
+        expect(valid).must_equal true
+      end
+
+    end
+
+    it 'requires a unique title in category' do
+      #other_book = book.clone
+      other_work = works(:pride)
+      other_work.title = work.title
+      other_work.category = work.category
+
+      valid = other_work.valid?
+
+      expect(valid).must_equal false
+      expect(other_work.errors.messages).must_include :title
+    end
+
+    it 'will allow a title that already exists if its in a different category' do
+      #other_book = book.clone
+      other_work = works(:pride)
+      other_work.title = work.title
+
+      valid = other_work.valid?
+
+      expect(valid).must_equal true
+    end
+  end
+
+  describe 'Media Page - Methods' do
+    it 'return top 10 of a given category' do
```

Comment:
You're not throughly testing all the methods here.  Just `top_movies` for 10 items and more than 10 and `top_books` for less than 10.  You don't test each.